### PR TITLE
Certifications section

### PR DESCRIPTION
**Pull request by kcbreuil**
- created certification card
- Created section
- have removed Section from About page, will push up after you have reviewed
Screen Shot 2021-06-15 at 11 39 17 AM

### CODE REVIEW

File: web/src/components/certificationSection.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Certification from './certification';
+
+export default function CertificationSection({ certifications }) {
+    return (
+        <div className="bg-brand-1 py-14 mb-2 text-white w-full">
```
Comment (asasmith) [inline web/src/components/certificationSection.js:7]: Do you need `mb-2`?

File: web/src/components/certification.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Certification({ data }) {
+    const { description, title } = data;
+
+    return (
+        <div className="odd:bg-white odd:bg-opacity-10 p-4 text-2xl">
```
Comment (asasmith) [inline web/src/components/certification.js:8]: Nice use of the `odd:` utility here!

**Review by asasmith — CHANGES_REQUESTED** [review]

File: web/src/components/certificationSection.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+            <div className="container grid grid-col gap-4">
+                <h2 className="text-ts-h2 uppercase">Certifications</h2>
```
[the remark this answers is not in the document]
Comment (kcbreuil) [reply web/src/components/certificationSection.js:9]: @asasmith i think this is a mistake actually, most headings are bolded. The designs are still pulling in Gallano and that is not correct

[the remark this answers is not in the document]
Comment (kcbreuil) [reply web/src/components/certificationSection.js:9]: @asasmith ok just confirmed, cross site we are keeping these specific headings unweighted. see news, history, leadership etc

**Review by asasmith — CHANGES_REQUESTED** [review]

**Pull request merged**